config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "GITHUB_TOKEN": "02c488529ccb97674c4655467617f0442f37cee6",
```

Comment:
It should be a read only access, I should verify that. This is a common github hack these days, hackers search repos for tokens for nefarious purposes. It may be best I delete this PR and re-submit a new one without that file in the history.

thanks for the advice!